packages/wallet-adapter-core/src/AIP62StandardWallets/registry.ts

```diff
@@ -0,0 +1,24 @@
+import { WalletName } from "../LegacyWalletPlugins/types";
+import { WalletReadyState } from "../constants";
+import { AptosStandardSupportedWallet } from "./types";
+
+/**
+ * Registry of AIP-62 wallet standard supported wallets.
+ * This list is used to show supported wallets even if they are not installed on the user machine.
+ *
+ * AIP-62 compatible wallets are required to add their wallet info here if they want to be detected by the adapter
+ *
+ * name - The wallet name
+ * url - The wallet website url (where users can download it from)
+ * icon - The wallet icon
+ */
+export const aptosStandardSupportedWalletList: Array<AptosStandardSupportedWallet> =
+  [
+    {
```

Comment:
Do we want to possibly look at making this decentralized in the future?  Or are we just hoping everyone contributes here.

(Or even let people customize which ones show up)

Reply:
Yea, we would probably modify it a bit in the future, for now we only have Nightly and we want it to show up for users so we go with this initial implementation